buildSrc/subprojects/buildquality/src/main/kotlin/org/gradle/gradlebuild/buildquality/CiReportingPlugin.kt

```diff
@@ -36,24 +36,30 @@ open class CiReportingPlugin : Plugin<Project> {
 
     private
     fun Project.prepareReportsForCiPublishing() {
-        val failedTaskGenericHtmlReports = failedTasks(allprojects).flatMap {
+        val executedTaskGenericHtmlReports = executedTasks(allprojects).flatMap {
             it.failedTaskGenericHtmlReports()
         }
         val attachedReports = executedTasks(subprojects).flatMap {
             it.attachedReportLocations()
         }
-        val failedTaskCustomReports = failedTasks(subprojects).flatMap {
+        val executedTaskCustomReports = executedTasks(subprojects).flatMap {
```

Comment:
I wonder if archiving the the custom reports always will take too much disk space.